TLS.md

```diff
@@ -85,18 +85,18 @@ but there are probably other good reasons to improve that part anyway.
 To-Do List
 ----------
 
-- [ ] redis-benchmark support. The current implementation is a mix of using
+- [ ] valkey-benchmark support. The current implementation is a mix of using
   hiredis for parsing and basic networking (establishing connections), but
   directly manipulating sockets for most actions. This will need to be cleaned
   up for proper TLS support. The best approach is probably to migrate to hiredis
   async mode.
-- [ ] redis-cli `--slave` and `--rdb` support.
+- [ ] valkey-cli `--slave` and `--rdb` support.
```

Comment:
After this, can you move these to issues instead of a random TODO in the TLS md.

Reply:
Sure, let me create 2 issues on this